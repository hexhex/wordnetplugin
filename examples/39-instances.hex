%% this returns all instances of a word

%% the search number for instances
%% is 39

word("actor").
instances(W,SW,I,SI) :- &wordnet[39,W](P,SW,I,SI),word(W).
