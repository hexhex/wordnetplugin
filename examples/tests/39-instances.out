{instances("actor",1,"George_Orson_Welles",72), instances("actor",1,"Orson_Welles",72), instances("actor",1,"Welles",72), instances("actor",1,"Duke_Wayne",71), instances("actor",1,"John_Wayne",71), instances("actor",1,"Wayne",71), instances("actor",1,"Peter_Alexander_Ustinov",70), instances("actor",1,"Sir_Peter_Ustinov",70), instances("actor",1,"Ustinov",70), instances("actor",1,"Sir_Herbert_Beerbohm_Tree",69), instances("actor",1,"Tree",69), instances("actor",1,"Spencer_Tracy",68), instances("actor",1,"Tracy",68), instances("actor",1,"Erich_von_Stroheim",67), instances("actor",1,"Stroheim",67), instances("actor",1,"Israel_Strassberg",66), instances("actor",1,"Lee_Strasberg",66), instances("actor",1,"Strasberg",66), instances("actor",1,"James_Maitland_Stewart",65), instances("actor",1,"Jimmy_Stewart",65), instances("actor",1,"Stewart",65), instances("actor",1,"Konstantin_Sergeevich_Alekseev",64), instances("actor",1,"Konstantin_Sergeyevich_Stanislavsky",64), instances("actor",1,"Konstantin_Stanislavsky",64), instances("actor",1,"Stanislavsky",64), instances("actor",1,"Otis_Skinner",63), instances("actor",1,"Skinner",63), instances("actor",1,"Francis_Albert_Sinatra",62), instances("actor",1,"Frank_Sinatra",62), instances("actor",1,"Sinatra",62), instances("actor",1,"Peter_Sellers",61), instances("actor",1,"Sellers",61), instances("actor",1,"George_C._Scott",60), instances("actor",1,"Scott",60), instances("actor",1,"Edward_Goldenberg_Robinson",59), instances("actor",1,"Edward_G._Robinson",59), instances("actor",1,"Robinson",59), instances("actor",1,"Sir_Ralph_David_Richardson",58), instances("actor",1,"Ralph_Richardson",58), instances("actor",1,"Richardson",58), instances("actor",1,"Charles_Robert_Redford",57), instances("actor",1,"Robert_Redford",57), instances("actor",1,"Redford",57), instances("actor",1,"Sidney_Poitier",56), instances("actor",1,"Poitier",56), instances("actor",1,"Peter_Seamus_O'Toole",55), instances("actor",1,"Peter_O'Toole",55), instances("actor",1,"O'Toole",55), instances("actor",1,"Baron_Olivier_of_Birghton",54), instances("actor",1,"Sir_Laurence_Kerr_Olivier",54), instances("actor",1,"Laurence_Olivier",54), instances("actor",1,"Olivier",54), instances("actor",1,"Paul_Leonard_Newman",53), instances("actor",1,"Paul_Newman",53), instances("actor",1,"Newman",53), instances("actor",1,"Dudley_Stuart_John_Moore",52), instances("actor",1,"Dudley_Moore",52), instances("actor",1,"Moore",52), instances("actor",1,"Robert_Mitchum",51), instances("actor",1,"Mitchum",51), instances("actor",1,"James_Neville_Mason",50), instances("actor",1,"James_Mason",50), instances("actor",1,"Mason",50), instances("actor",1,"Steve_Martin",49), instances("actor",1,"Martin",49), instances("actor",1,"E._G._Marshall",48), instances("actor",1,"Marshall",48), instances("actor",1,"Alfred_Lunt",47), instances("actor",1,"Lunt",47), instances("actor",1,"Bela_Ferenc_Blasko",46), instances("actor",1,"Bela_Lugosi",46), instances("actor",1,"Lugosi",46), instances("actor",1,"Laszlo_Lowestein",45), instances("actor",1,"Peter_Lorre",45), instances("actor",1,"Lorre",45), instances("actor",1,"Harold_Clayton_Lloyd",44), instances("actor",1,"Harold_Lloyd",44), instances("actor",1,"Lloyd",44), instances("actor",1,"John_Uhler",43), instances("actor",1,"Jack_Lemmon",43), instances("actor",1,"Lemmon",43), instances("actor",1,"Lee_Yuen_Kam",42), instances("actor",1,"Bruce_Lee",42), instances("actor",1,"Lee",42), instances("actor",1,"Charles_Laughton",41), instances("actor",1,"Laughton",41), instances("actor",1,"Eugene_Curran_Kelly",40), instances("actor",1,"Gene_Kelly",40), instances("actor",1,"Kelly",40), instances("actor",1,"Joseph_Francis_Keaton",39), instances("actor",1,"Buster_Keaton",39), instances("actor",1,"Keaton",39), instances("actor",1,"Edmund_Kean",38), instances("actor",1,"Kean",38), instances("actor",1,"William_Henry_Pratt",37), instances("actor",1,"Boris_Karloff",37), instances("actor",1,"Karloff",37), instances("actor",1,"Asa_Yoelson",36), instances("actor",1,"Al_Jolson",36), instances("actor",1,"Jolson",36), instances("actor",1,"Leslie_Howard_Stainer",35), instances("actor",1,"Leslie_Howard",35), instances("actor",1,"Howard",35), instances("actor",1,"Sir_Anthony_Philip_Hopkins",34), instances("actor",1,"Sir_Anthony_Hopkins",34), instances("actor",1,"Anthony_Hopkins",34), instances("actor",1,"Hopkins",34), instances("actor",1,"Dustin_Hoffman",33), instances("actor",1,"Hoffman",33), instances("actor",1,"John_Hemminge",32), instances("actor",1,"John_Heming",32), instances("actor",1,"Hemminge",32), instances("actor",1,"Heming",32), instances("actor",1,"Reginald_Carey_Harrison",31), instances("actor",1,"Sir_Rex_Harrison",31), instances("actor",1,"Rex_Harrison",31), instances("actor",1,"Harrison",31), instances("actor",1,"Thomas_J._Hanks",30), instances("actor",1,"Tom_Hanks",30), instances("actor",1,"Hanks",30), instances("actor",1,"Sir_Alec_Guinness",29), instances("actor",1,"Alec_Guinness",29), instances("actor",1,"Guinness",29), instances("actor",1,"Harley_Granville-Barker",28), instances("actor",1,"Granville-Barker",28), instances("actor",1,"Cary_Grant",27), instances("actor",1,"Grant",27), instances("actor",1,"Arthur_John_Gielgud",26), instances("actor",1,"Sir_John_Gielgud",26), instances("actor",1,"Gielgud",26), instances("actor",1,"Mel_Columcille_Gerard_Gibson",25), instances("actor",1,"Mel_Gibson",25), instances("actor",1,"Gibson",25), instances("actor",1,"David_Garrick",24), instances("actor",1,"Garrick",24), instances("actor",1,"William_Clark_Gable",23), instances("actor",1,"Clark_Gable",23), instances("actor",1,"Gable",23), instances("actor",1,"Henry_Fonda",22), instances("actor",1,"Fonda",22), instances("actor",1,"Douglas_Fairbanks_Jr.",21), instances("actor",1,"Julius_Ullman",20), instances("actor",1,"Douglas_Elton_Fairbanks",20), instances("actor",1,"Douglas_Fairbanks",20), instances("actor",1,"Fairbanks",20), instances("actor",1,"Fairbanks",21), instances("actor",1,"John_Drew",19), instances("actor",1,"Drew",19), instances("actor",1,"Gerard_Depardieu",18), instances("actor",1,"Depardieu",18), instances("actor",1,"Robert_De_Niro",17), instances("actor",1,"De_Niro",17), instances("actor",1,"James_Byron_Dean",16), instances("actor",1,"James_Dean",16), instances("actor",1,"Dean",16), instances("actor",1,"Harry_Lillis_Crosby",15), instances("actor",1,"Bing_Crosby",15), instances("actor",1,"Crosby",15), instances("actor",1,"Hume_Blake_Cronyn",14), instances("actor",1,"Hume_Cronyn",14), instances("actor",1,"Cronyn",14), instances("actor",1,"Sir_Noel_Pierce_Coward",13), instances("actor",1,"Noel_Coward",13), instances("actor",1,"Coward",13), instances("actor",1,"Frank_Cooper",12), instances("actor",1,"Gary_Cooper",12), instances("actor",1,"Cooper",12), instances("actor",1,"Maurice_Chevalier",11), instances("actor",1,"Chevalier",11), instances("actor",1,"James_Cagney",10), instances("actor",1,"Jimmy_Cagney",10), instances("actor",1,"Cagney",10), instances("actor",1,"Richard_Burton",9), instances("actor",1,"Burton",9), instances("actor",1,"Richard_Burbage",8), instances("actor",1,"Burbage",8), instances("actor",1,"John_Wilkes_Booth",7), instances("actor",1,"Booth",7), instances("actor",1,"Humphrey_DeForest_Bogart",6), instances("actor",1,"Humphrey_Bogart",6), instances("actor",1,"Bogart",6), instances("actor",1,"John_Barrymore",5), instances("actor",1,"Lionel_Barrymore",4), instances("actor",1,"Herbert_Blythe",3), instances("actor",1,"Maurice_Barrymore",3), instances("actor",1,"Barrymore",3), instances("actor",1,"Barrymore",4), instances("actor",1,"Barrymore",5), instances("actor",1,"Fred_Astaire",2), instances("actor",1,"Astaire",2), instances("actor",1,"Allen_Stewart_Konigsberg",1), instances("actor",1,"Woody_Allen",1), instances("actor",1,"Allen",1), word("actor")}
